config/manager/manager.yaml

```diff
@@ -30,6 +30,8 @@ spec:
           value: "registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:centos8-1.16-0"
         - name: RELATED_IMAGE_PGEXPORTER
           value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.0.4-0"
+        - name: RELATED_IMAGE_PGUPGRADE
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-upgrade:centos8-14.1-0"
```

Comment:
~Should this match the postgres/gis versions of 13.5 :question:~ ...Actually why do we have both 13 and 14 versions here :confused:

Reply:
I believe the goal was to show the two most recent versions, since the PG version values are included. As for the new `RELATED_IMAGE_PGUPGRADE` value, that can use the latest version because the previous versions are included to facilitate the various upgrades.

Reply:
Can we update any `crunchy-upgrade` tags per sc-12985?  E.g.

```
registry.developers.crunchydata.com/crunchydata/crunchy-upgrade:crunchy-upgrade:ubi8-5.1.0-0
```

We would also want to do the same for `installers/olm/config/redhat/related-images.yaml ` below.

Reply:
Sure thing, I'll make those updates.